tools/qontract_cli.py

```diff
@@ -517,6 +517,15 @@ def get_available_upgrades(ocm: OCM, version: str, channel: str) -> list[str]:
     return [u for u in upgrades if not ocm.version_blocked(u)]
 
 
+def inheritVersionDataText(ocm_org: str, ocm_specs: list[dict]) -> str:
```

Comment:
would you consider snake case as the pythonic way of naming things?

Reply:
oh, sure

Reply:
Fixed in eb4f8e51